assets/src/scss/layout/navbar/_navigation-mobile-menu.scss

```diff
@@ -0,0 +1,223 @@
+#navigation-mobile-menu {
+  position: absolute;
+  display: flex;
```

Comment:
I don't think this needs to be using Flexbox, it seems to look exactly the same if you remove this line and the two following ones 🤔

Reply:
The idea of using `flexbox` is because I need to make a "tricky" implementation since the Design team has requested to have the Donate Button always sticky in the bottom. 

I found a solution to set the `display` to `flex` with a `space between` direction to the main container, and then set  to the menu items with a `100%` of `height`. 

If we have a long list of items, then we could scroll it and doing this we'll have always the donate button sticky in the bottom menu avoiding to use `position: fixed|absolute` to the button.

Does it make sense?

Reply:
Aaaah okay yes I see! Nevermind 😁